### PR TITLE
fix(components): add separator in message tx that contains token

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/components/action-msg/SingleMsg.tsx [hunk 1]
```diff
@@ -43,11 +43,15 @@ export const SingleMsg = ({
...
       </Text>
       {tokens?.map((token: TokenWithValue, index: number) => (
-        <MsgToken
-          key={index.toString() + token.denom}
-          token={token}
-          // TODO: add `ampCopierSection` later
-        />
+        <Flex>
+          <MsgToken
+            key={index.toString() + token.denom}
```
Comment (Poafs1) [inline src/lib/components/action-msg/SingleMsg.tsx:48]: we no need to add `index.toString()`. Just `key={token.denom}` should be enough

File: src/lib/components/action-msg/SingleMsg.tsx [hunk 2]
```diff
@@ -43,11 +43,15 @@ export const SingleMsg = ({
...
+            token={token}
+            // TODO: add `ampCopierSection` later
+          />
+          {index < tokens.length - 2 && <Text>, </Text>}
+          {index === tokens.length - 2 && <Text ml={1}>and</Text>}
```
Comment (Poafs1) [inline src/lib/components/action-msg/SingleMsg.tsx:53]: `{index < tokens.length - 2 ? <Text>,</Text> : <Text ml={1}>and</Text>}`

**Review by Poafs1 — COMMENTED** [review]
lgtm, just few comments

File: src/lib/components/action-msg/SingleMsg.tsx [hunk 3]
```diff
@@ -43,11 +43,14 @@ export const SingleMsg = ({
         {type} {text1}
       </Text>
       {tokens?.map((token: TokenWithValue, index: number) => (
-        <MsgToken
-          key={index.toString() + token.denom}
-          token={token}
-          // TODO: add `ampCopierSection` later
-        />
+        <Flex>
```
Comment (evilpeach) [inline src/lib/components/action-msg/SingleMsg.tsx:46]: Move key from line 48 to this line

**Review by evilpeach — APPROVED** [review]

**Pull request merged**